tools/dev/install-dev-tools.sh

```diff
@@ -8,11 +8,14 @@ TOOLS_DEPS_DIRS="$3"
 TOOLS_DEPS_LOCK_FILE="$4"
 GOOS="$5"
 GOARCH="$6"
+# We cannot provide list of arguments not as a string, so we join them with a comma
+TOOLS_DEPS_DIRS=${TOOLS_DEPS_DIRS//,/ }
 
 mkdir -p "$CI_TOOLS_BIN_DIR" "$CI_TOOLS_DIR"/protos
+read -ra TOOLS_DEPS_DIRS <<< "${TOOLS_DEPS_DIRS[@]}"
```

Comment:
I suspect you can prefix this with `IFS=","` instead of doing the parameter replace thing above